File: adapters/maze-service/src/main/java/sk/slsp/workshop/mazesolver/adapters/mazeservice/configuration/MazeServiceAdapterConfiguration.java
package sk.slsp.workshop.mazesolver.adapters.mazeservice.configuration;

import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;
import sk.slsp.workshop.mazesolver.adapters.mazeservice.adapter.MazeServiceAdapter;

@Configuration
public class MazeServiceAdapterConfiguration {
	
	@Bean
	public MazeServiceAdapter mazeServiceAdapter() {
		return new MazeServiceAdapter();
	}
	
}

File: adapters/maze-service/src/main/kotlin/sk/slsp/workshop/mazesolver/adapters/mazeservice/configuration/MazeServiceAdapterConfiguration.kt
package sk.slsp.workshop.mazesolver.adapters.mazeservice.configuration

import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration
import sk.slsp.workshop.mazesolver.adapters.mazeservice.adapter.MazeServiceAdapter

@Configuration
class MazeServiceAdapterConfiguration {

	@Bean
	fun mazeService() =
		MazeServiceAdapter()

}
